modules/power-monitoring-deploying-kepler-redfish.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+
+// * power_monitoring/installing-power-monitoring.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="power-monitoring-deploying-kepler-redfish_{context}"]
+= Deploying {PM-kepler} using Redfish API
+
+You can use the Redfish API as the source to deploy {PM-kepler}. Redfish API increases the accuracy and efficiency of {PM-kepler} under certain circumstances.
+
+.Prerequisites
+* You have access to the {product-title} web console.
+* You are logged in as a user with the `cluster-admin` role.
+* You have installed the {PM-operator}.
+
+.Procedure
+
+. In the *Administrator* perspective of the web console, go to *Operators* -> *Installed Operators*.
+
+. Click *{PM-title-c}* from the *Installed Operators* list and go to the *{PM-kepler}* tab.
+
+. Click *Create {PM-kepler}*.
+
+. Create a {PM-kepler} instance with `.spec.exporter.redfish` specifying the `secretRef`. `secretRef` refers to the name of secret which contains credentials to initialize `RedfishClient`.
++
+[NOTE]
+====
+The `openshift-power-monitoring` namespace is created after {PM-kepler} is deployed.
+====
+
+. Create the secret using `openshift-power-monitoring` namespace.
++
+[source,terminal]
+----
+$ oc -n openshift-power-monitoring create secret generic redfish-secret --from-file=redfish.csv
```

Comment:
What should be in redfish.csv?  Should we provide an example?

Reply:
+1 We should. Otherwise it is very hard to guess. 

@shreyasiddhartha could you please add the information provide in the jira 

```
redfish.csv Data Format:

your_kubelet_node_name, redfish_username, redfish_password, https://redfish_ip_or_hostname/

Example redfish.csv

kind-control-plane,dummyuser,dummypass,https://dummy-machine.lab.eng.redhat.com
kind-worker,dummyuser,dummypass,https://dummy-machine.lab.eng.redhat.com
kind-worker2,dummyuser,dummypass,https://dummy-machine.lab.eng.redhat.com
```